test/winmain.cpp

```diff
@@ -40,7 +38,7 @@ void test_winmain()
 #define C ,
 #define TEST(input, expected) \
     char* BOOST_PP_CAT(e, __LINE__)[] = expected;\
-    vector<string> BOOST_PP_CAT(v, __LINE__) = split_winmain(input);\
+    std::vector<std::string> BOOST_PP_CAT(v, __LINE__) = split_winmain(input);\
```

Comment:
Likewise.

Reply:
Please, this is C++. You really want to have 'using namespace std;', include C++ and C header files, and leave off the std:: namespaces everywhere you can ? All that together can lead to problems, and did wirth the Embarcadero C++ compiler. The code already produces warnings on the invocation of TEST, which I did not bother to change, trying to keep my changes to a minimum. How am I hurting your code by providing the std:: namespace excplicitly ? It is not breaking anything and is certainly avoiding any other potential problems.

Reply:
All this is valid C++ that no other compiler had problem with for a decade. I think it's reasonable to ask what specific problem Borland had here

Reply:
The specific problem Embarcadero had is that there is a `std::byte` ( https://en.cppreference.com/w/cpp/types/byte ) and there is an unqualified `byte` in the Windows header file rpcndr.h. Having `using namespace std;` creates the clash. Before you ask why Embarcadero encounters this problem while other compilers do not, let me answer that I did not want to spend hours trying to figure this out rather than spend minutes fixing it with perfectly reasonable code which changes nothing as far as the logic goes. Really Vladimir, you know as well as I do that specifying `using namespace std;` is generally a poor choice for anything but the simplest code which #includes very little and using any Boost library's code is never that simple.

Reply:
I fixed the test/winmain.cpp in a better way without the previous changes I had made.